bazel/Dockerfile

```diff
@@ -49,6 +49,7 @@ RUN \
 
 # Store the Bazel outputs under /workspace so that the symlinks under bazel-bin (et al) are accessible
 # to downstream build steps.
+RUN mkdir /workspace
```

Comment:
maybe `mkdir -p /workspace` in case it already exists?

Reply:
(done)